packages/2d/src/curves/CircleSegment.ts

```diff
@@ -47,10 +47,16 @@ export class CircleSegment extends Segment {
     const endTangent = Vector2.fromRadians(endAngle);
 
     return [
-      this.center.add(startTangent.scale(this.radius)),
-      this.counter ? startTangent : startTangent.flipped,
-      this.center.add(endTangent.scale(this.radius)),
-      this.counter ? endTangent.flipped : endTangent,
+      {
+        position: this.center.add(startTangent.scale(this.radius)),
+        tangent: this.counter ? startTangent : startTangent.flipped,
+        normal: this.counter ? startTangent : startTangent.flipped,
+      },
+      {
+        position: this.center.add(endTangent.scale(this.radius)),
+        tangent: this.counter ? endTangent.flipped : endTangent,
+        normal: this.counter ? endTangent.flipped : endTangent,
```

Comment:
I think we should also rename the variables to better reflect what they are,
```suggestion
        position: this.center.add(startTangent.scale(this.radius)),
        tangent: this.counter ? startNormal : startNormal.flipped,
        normal: this.counter ? startNormal : startNormal.flipped,
      },
      {
        position: this.center.add(endTangent.scale(this.radius)),
        tangent: this.counter ? endNormal.flipped : endNormal,
        normal: this.counter ? endNormal.flipped : endNormal,
```